core/client/src/main/java/org/phenotips/remote/client/internal/RemoteMatchFinder.java

```diff
@@ -82,59 +82,65 @@ public int getPriority()
     }
 
     @Override
-    protected Set<String> getSupportedServerIdList()
+    public Set<String> getSupportedServerIdList()
     {
         return this.getRemotesList();
     }
 
     @Override
-    protected MatchRunStatus specificFindMatches(Patient patient, String remoteId, List<PatientMatch> matchesList)
+    protected Response specificFindMatches(Patient patient, String remoteId, List<PatientMatch> matchesList)
```

Comment:
The same comment about returning a `Response` in PN code applies here - and should be kept as is or resolved the same way as in PN

Reply:
Will leave for future  development